### PR TITLE
Add Member CI/CD

### PR DESCRIPTION
**Pull request by smooth-dasilva**

### CODE REVIEW

**Review by leandroyabut — COMMENTED** [review]
There are some things missing in my package.json file. Was there anything that you needed to remove? I also see that some of the versions were downgraded as well. Was that for a specific reason as well?

**Comment by smooth-dasilva** [comment]
Yeah sorry about that not sure how I managed to do that but it wasn't manually. I've fixed the issue in the recent commit.

File: karma.conf.js [hunk 1]
```diff
@@ -25,20 +34,31 @@ module.exports = function (config) {
...
     },
-    reporters: ['progress', 'kjhtml'],
+    reporters: ['progress', 'sonarqubeUnit', 'kjhtml'],
     port: 9876,
     colors: true,
     logLevel: config.LOG_INFO,
     autoWatch: true,
-    browsers: ['Chrome'],
+    restartOnFileChange: true,
+    browsers: ['ChromeHeadless'],
```
Comment (leandroyabut) [inline karma.conf.js:50]: Hey Luan, how would this affect my local tests? Will my Karma server and UI still run?

Comment (smooth-dasilva) [reply karma.conf.js:50]: I think it would run fine, but should I add both to the list to make sure?

**Pull request closed without merging**